### ISSUE TITLE
Random crash upon entering the basement

### ISSUE THREAD

**Issue by DarkDaskin** [a1studmuffin/Cataclysm-DDA-Android#57]
Build 0.C-26322

Most often it crashes when I enter the basement. Sometimes it does not, but then it crashes when I kill a zombie there.

Tried the same save game on Windows version (same build) and got no problems even after multiple reloads.

[Save.zip](https://github.com/a1studmuffin/Cataclysm-DDA-Android/files/1745011/default.zip)
[crash.txt](https://github.com/a1studmuffin/Cataclysm-DDA-Android/files/1745014/crash.txt)

**Comment by StopGamer** [comment]
I tried different new worlds and get same issue.
Only one exception - I loaded save before update with new character and was able to go down and even kill bloated zombie. After I created second one character and was able to go down again.

Anyway in old and new worlds I got crashes trying to get to the nearest city. And old character is unloadable due being saved in center of the city. Not sure if this connected to this issue.
